### ISSUE TITLE
Add 2FA

### ISSUE THREAD

**Issue by WaveHack** [WaveHack/OpenDominion#65]
Low low prio, but it's on the list so I won't forget it

**Comment by MJBrennan** [comment]
Hi this I could help out with if you want? I've been a lot of 2FA recently

**Comment by WaveHack** [comment]
I have some plans for security related things, but I'd like to do these myself to get more familiar with implementing them (like 2FA), if you don't mind. 😃

**Comment by Dr-Eki** [comment]
Before doing this, consider the value of what's being protected vs. the frustration of losing access to your account because you lost your 2FA device, plus your time to help users re-access their account. You might set yourself up for a lot of support work and abandoned player accounts because people change phones.

**Comment by WaveHack** [comment]
True. Although 2FA would be optional and opt-in for people who like the extra security.
